content/applications/general.rst

```diff
@@ -4,12 +4,13 @@ Miscellaneous
 
 .. toctree::
 
-    general/users
-    general/auth
     general/apps_modules
-    general/export_import_data
-    general/payment_acquirers
-    general/voip
+    general/auth
     general/calendars
-    general/in_app_purchase
     general/developer_mode
+    general/digest_emails
+    general/export_import_data
+    general/in_app_purchase
+    general/payment_acquirers
+    general/users
+    general/voip
```

Comment:
Hi @jcs-odoo 

Thank you for all of the suggestions. I'll work on these and get another version pushed up this week 👍 

As for the reorganization piece, I see where you're coming from. 

I'd like to entertain a separate PR in the future that attempts to reorganize all of the sections alphabetically, with the assumption that this type of organization would be helpful in eliminating friction for the end user (e.g., there's already friction in digesting/implementing the information in the documentation without the added burden of taking the time to search for the correct information).

I see two progressive options we could pursue to produce a _fair_ + _ideal_ working model.
- The _fair_ model uses  "business English" as the leading language as this is the default language set for `/`.
- The _ideal_ model might leverage an extension or custom script inside `conf.py` to accommodate for separate languages, organizing by the first character in the title (something like [this](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_member_order) maybe).

Can circle back on it later if you're open to either of these ⏳

Reply:
Hello @StraubCreative and @jubodoo 

As I mentioned in my previous comment, I agree that a reorganization is needed, but I also think that a better reorganization would be _outside_ the misc section. I don't have any good suggestion for this right now, though.

I'm not against sorting the categories of the misc section alphabetically, but that would be the only place in the doc where that logic could work, and only for business English, as you pointed out. If you do so, it wouldn't be in this commit about the new page "digest emails" and certainly in another branch based on 13.0 rather than 15.0.

The issue here is that it is a "misc" section. Because there are many different things thrown together in that section, there is no reason to have one category or page before another. So indeed, sorting the misc categories alphabetically could partially work here for that level, but it would also put more "noise" in the structure. For example, the pages 
- In-App Purchase (IAP)
- Developer Mode (debug mode)
were meant to be at the end of the misc section because they are mostly referenced in other doc pages or in the apps themselves. We don't expect users to actually browse the doc structure looking for these two pages.

Also, it wouldn't work per se for the children pages of these categories. For example, here would be the alphabetical order of the children pages of misc/users. "Manage users" which is meant to be the first page of this category would come last:
- Users & Companies
  - Access Rights
  - Change language
  - Companies
  - Manage users

In my opinion, an alphabetical order would certainly not work for the other sections of the documentation.
Take this Websites section for example.
![image](https://user-images.githubusercontent.com/46810843/149473148-764a8821-ec02-4544-b4e5-385ee928ae71.png)
Sorting everything in alphabetical order would be a non-sense as "Website" would come last, and the order of the eCommerce categories would make no sense either: 

- Websites
  - eCommerce
    - Collect taxes
    - Get paid
    - Get started
    - Launch my website
    - Manage my products
    - Maximize my revenue
  - Live Chat
  - Website

That's why I commented that, in my opinion, it is not a good answer to the problem, even just for the misc section. I'm not blocking it either, of course. At least you have my detailed opinion :) 

Have a good one!

Jonathan